### PR TITLE
Fix: dust app params generations check

### PR DESCRIPTION
**Pull request by Fraggle**
## Description

This PR fixes an edge case in the Dust app parameters generation check process. 

Changes:
- File: `front/lib/api/assistant/actions/dust_app_run.ts`
- 1 line modified

Improvements:
- Better handling of edge cases (empty, null, undefined, 0, false)

## Risk

Low risk due to minimal change. Main concern is potential unexpected behavior with certain parameter inputs.

## Deploy

1. Test thoroughly (unit and integration tests)
2. Deploy to staging and verify
3. Monitor logs and error rates during/after production deployment
4. Have rollback plan ready (should be simple given the small change)

### CODE REVIEW

File: front/lib/api/assistant/actions/dust_app_run.ts [hunk 1]
```diff
@@ -237,7 +237,7 @@ export class DustAppRunConfigurationServerRunner extends BaseActionConfiguration
     const params: DustAppParameters = {};
 
     for (const k of spec.inputs) {
-      if (rawInputs[k.name] && typeof rawInputs[k.name] === k.type) {
+      if (k.name in rawInputs && typeof rawInputs[k.name] === k.type) {
```
Comment (fontanierh) [inline front/lib/api/assistant/actions/dust_app_run.ts:240]: Second condition should be enough no? If undefined it will be false

**Review by fontanierh — APPROVED** [review]
lgtm

File: front/lib/api/assistant/actions/dust_app_run.ts (hunk 1 above)
Comment (Fraggle) [reply front/lib/api/assistant/actions/dust_app_run.ts:240]: I think you are correct.
Let's keep the first check, it's more explicit.

**Review by spolu — COMMENTED** [review]
LGTM
